donjo-app/controllers/Penduduk.php

```diff
@@ -276,6 +276,7 @@ public function delete_all_dokumen($id_pend = 0)
 
 	public function cetak_biodata($id = '')
 	{
+		$header = $this->header_model->get_data();
 		$data['desa'] = $header['desa'];
```

Comment:
Apa bisa ditulis bgni pak ???

$data['desa'] = $this->_header['desa'];

Reply:
Bisa karena didefine di private $_header; tapi kalau tidak diganti jangan dihapus dulu script aslinya.

Reply:
ya